swap_meet/vendor.py

```diff
@@ -1,2 +1,58 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        inventory = inventory if inventory is not None else []
+        self.inventory = inventory
```

Comment:
Nice handling of the None default value using a Ternary operator! I generally stay away from overwriting the data we get from our parameters in case we ever need that info back. In this case there's no harm, but if we wanted to avoid overwriting the `inventory` parameter we could use the ternary expression to directly assign `self.inventory`'s value:
```suggestion
        self.inventory = inventory if inventory is not None else []
```